content/millennium/dstu2/medications/medication-order.md

```diff
@@ -75,11 +75,12 @@ _Implementation Notes_
 
  Name                   | Required? | Type          | Description
 ------------------------|-----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- `patient`              | Y         | [`reference`] | The identity of a patient to list orders for. Example: `12345`
- `status`               | N         | [`token`]     | The status of the medication order, may be a list separated by commas. Example: `active,draft`
- `-timing-boundsPeriod` | N         | [`date`]      | The date-time which should fall within the [period] the medication should be given to the patient. Must be prefixed by 'ge'. Example: `ge2014-05-19T20:54:02.000Z`
- `_lastUpdated`         | N         | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication. Must be prefixed by 'ge' or 'le'. Example: `ge2014-05-19T20:54:02.000Z`
- [`_count`]             | N         | [`number`]    | The maximum number of results to include in a page. Example: `50`
+ `_id`                  | Y or `patient` | [`token`]     | The unique identifier representing the medication order. Example: `12345`
```

Comment:
Every other resource describes this as "The logical resource id associated with the <resource type here>". Any reason not to go with that? An "identifier" is not the same thing as an "id" in FHIR so it's best not to use them interchangeably

Reply:
I had actually copied this from the Conformance statement. I've changed it to match everywhere else: https://github.com/cerner/fhir.cerner.com/pull/217/commits/8480c0a3b714751c5e770f303c1da69e238d4483